github_api/voting.py

```diff
@@ -160,21 +161,29 @@ def parse_review_for_vote(state):
 
 def parse_reaction_for_vote(body):
     """ turns a comment reaction into a vote, if possible """
-    try:
```

Comment:
I'm not sure that anything that happened here qualifies as "rigorous code review". Lol. I'd guess it was more likely to be a genuine mistake than actively malicious.